sync-mangle.js

```diff
@@ -0,0 +1,43 @@
+const { resolve } = require('path');
+const { readFileSync, writeFileSync } = require('fs');
+
+function readAsJSON(relativePath) {
+	return JSON.parse(readFileSync(resolve(__dirname, relativePath), 'utf-8'));
+}
+
+function writeAsJSON(relativePath, body) {
+	return writeFileSync(
+		resolve(__dirname, relativePath),
+		JSON.stringify(body, null, 2)
+	);
+}
+
+// Get the root file
+const root = readAsJSON(resolve(__dirname, 'mangle.json'));
+
+// List the other mangles
+const children = [
+	'./compat/mangle.json',
+	'./debug/mangle.json',
+	'./hooks/mangle.json'
+];
+
+// Go through each file
+children.forEach(childPath => {
```

Comment:
Could we simply use _exactly_ the same mangle file for all packages? Then the sync script could just do an `fs.copyFileSync` for each source/destination path pair.

Reply:
I like how you're thinking! The only thing that didn't made me do a straight copy is that nested packages do not have all the properties from root mangle. And then there's other properties that might be customised?

But if my thinking is wrong we could totally just copy them in there.

Reply:
That's true today, but once the next microbundle is released there will only be one `mangle.json` file that they all will share. So any customization will go away and they will all be in the one root `mangle.json`. So perhaps we should just coalesce the `mangle.json`s now?

Reply:
Awesome! Then, that confirms @robertknight suggestion and I'll copy the files directly. Thanks for jumping in!